src/style.css

```
@@ -1 +1,87 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,200&family=Roboto:wght@500&display=swap");
+
+body,
+nav,
+.list {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
```

```suggestion
* { 
  box-sizing: border-box;
}

body,
.nav,
.list {
  margin: 0;
  padding: 0;
}

html {
  font-family: "Roboto", Helvetica, sans-serif;
}
```
Let's improve and structurize a little bit) 

1. You should not set properties in `*` selector because it can affect the page performance, but in the case of `box-sizing `it's ok, it's better when all elements have same `box-sizing`
2. Better to set `font-family` to `html`